docs/source/Plugin/P093_commands.repl

```diff
@@ -32,3 +32,8 @@
         ","
         Value = ``<<``, ``<``, ``|``, ``>``, ``>>``, ``<>`` or ``SWING``
         "
+        "
+        ``MitsubishiHP,remotetemperature,<value>``
+        ","
+        Value equal ``0.0`` uses internal Temperature Sensor and greater than ``0.0`` external Temperature Sensor .
```

Comment:
Maybe use same wording here?  (I like the use of backticks for the numbers by the way)